### ISSUE TITLE
Close button is not keyboard accessible while using screen reader

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#250]
**Description**
Pressing enter on the close button of the sidekick extension toolbar while using screen reader does not close the menu. In order to close it, the user have to use only mouse click.

**To Reproduce**
Steps to reproduce the behavior using screen reader (NVDA):
1. Open the URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Press Alt key to go to the chrome menu and press tab to go to the sidekick extension 
3. Press enter on the sidekick extension
4. Press tab to go to the close button and press enter to close the menu. 

**Expected behavior**
Close button is an interactive element of the page and it should work with the keyboard while using screen reader

**Screenshots**
Attached is the screenshot and the screen reader text. It reads close button but is not clickable

**Additional context**
https://www.w3.org/TR/WCAG21/#keyboard

**Why it matters**
Users can't access a web app's functionality if they can't access its interactive interface components. Many people, including those who are blind or who have low vision or hand tremors, rely on a keyboard (or keyboard alternate) to access those components.

![Close button not keyboard accessible 3](https://user-images.githubusercontent.com/123964084/218698079-1112b748-09a1-4d2e-ba5b-bc75901822fc.png)

**Comment by rofe** [comment]
I can't reproduce this.

**Comment by puja-iaccessible** [comment]
I am attaching a video. It might be helpful.
[Close button with screen reader.webm](https://user-images.githubusercontent.com/123964084/219085791-00206212-cda7-42a4-af5c-ba1f49f5da3d.webm)

**Comment by rofe** [comment]
@puja-iaccessible thanks for the recording. Unfortunately, I still can't reproduce it. I don't have a screen reader installed, but when I tab to the close button and hit return, the sidekick is closing as expected. The close function is registered as a `click` listener on the button element, which also seems to be triggered via return keystroke. Could it be that the screen reader interferes with that browser default behavior?

[Using Sidekick close button via keyboard.webm](https://user-images.githubusercontent.com/1609742/223818469-d977e7e8-3577-4a83-8245-b5c0670a0e85.webm)

**Comment by puja-iaccessible** [comment]
@rofe, Please close the ticket as I am unable to replicate it now. It is working with enter button using the screen reader now.
